app/Http/Controllers/UserController.php

```diff
@@ -172,8 +172,8 @@ public function delete(Request $request, string $id): View
     {
         if($request->user()->isAdmin())
```

Comment:
This should be `Auth::user`.

Reply:
resolved in 59ff09f7743b739de37cce6a45b1e3a7053d17fb